include/fluent-bit/flb_gzip.h

```diff
@@ -21,7 +21,8 @@
 #define FLB_GZIP_H
 
 #include <fluent-bit/flb_info.h>
-#include <stdio.h>
+#include <monkey/mk_http.h>
+#include <stdbool.h>
```

Comment:
We need to compile fluent-bit in ancient CenOSs. So, could you remove the stdbool.h dependency?